src/NHibernate/Id/IPostInsertIdentityPersister.cs

```diff
@@ -31,6 +104,8 @@ public interface IPostInsertIdentityPersister
 		/// column(s) to use in the select statement restriction.
 		/// </param>
 		/// <returns> The SQL select string </returns>
+		// Since 5.2
+		[Obsolete("Have the persister implement IMultiPropertiesPostInsertIdentityPersister and use its GetSelectByUniqueKeyString(string[] suppliedPropertyNames, out IType[] parameterTypes).")]
```

Comment:
The comment has to change to a new name

Reply:
Done.

(And rebased, commits directly amended.)